test/e2e/apps/job.go

```diff
@@ -83,8 +83,13 @@ var _ = SIGDescribe("Job", func() {
 		framework.ExpectNoError(err, "failed to get PodList for job %s in namespace: %s", job.Name, f.Namespace.Name)
 	})
 
-	// Pods sometimes fail, but eventually succeed.
-	ginkgo.It("should run a job to completion when tasks sometimes fail and are locally restarted", func() {
+	/*
+		Release : v1.16
+		Testname: Jobs, completion after task failure
+		Description: Explicitly cause the tasks to fail once initially. After restarting, the Job MUST
+		execute to completion and should be successful.
```

Comment:
should be successful -> MUST be successful? Or must remove that "and should be..."?

Reply:
Please address this

Reply:
Please address this

Reply:
-> Removed the "and should be successful" part